content/03.results.md

```diff
@@ -4,10 +4,32 @@
-Next, we created a public Github analysis repository (https://github.com/AlexsLemonade/OpenPBTA-analysis) with continuous integration to ensure analysis reproducibility.
-We additionally created a GitHub manuscript repository (https://github.com/AlexsLemonade/OpenPBTA-manuscript) with ManuBot [@url:https://manubot.org/ ; @doi:10.1371/journal.pcbi.1007128] integration to enable manuscript creation using Markdown within GitHub.
-
-We maintained a data release folder on Amazon S3 containing merged files for each analysis, downloadable from the GitHub repository or open access CAVATICA project.  
+We maintained a data release folder on Amazon S3 containing merged files for each analysis, downloadable from the GitHub repository or open access CAVATICA project.
+As new analytical products that we expected to be used across multiple analyses were produced (e.g., tumor mutation burden calculations) or issues with the data were identified, new releases were made available in a versioned manner.
+
+A key feature of this project was the open contribution model used for both analyses (i.e., analytical code) and scientific manuscript writing. 
+We created a public Github analysis repository (https://github.com/AlexsLemonade/OpenPBTA-analysis) to hold all code associated with analyses downstream of the Kids First Data Resource Center workflows and a GitHub manuscript repository (https://github.com/AlexsLemonade/OpenPBTA-manuscript) with ManuBot [@url:https://manubot.org/; @doi:10.1371/journal.pcbi.1007128] integration to enable manuscript creation using Markdown within GitHub.
+
+
+Additions to the manuscript often started with proposing an analysis in the analysis repository.
+A potential contributor could propose an analysis by filing an issue on the GitHub analysis repository. 
+Organizers for the project or other contributors with expertise had the opportunity to comment on the proposal (**Figure {@fig:Fig1}C**). 
+The contributor then forked the analysis repository (or make a copy for the purpose of adding code changes) and added their proposed analysis to their fork. 
+They then filed a pull request on GitHub, where they requested to add their proposed changes to the main repository. 
+All pull requests to the analysis repository underwent peer review by organizers or other contributors. 
+Analytical code reviews conducted by peer reviewers checked for correctness of analyses (**Figure {@fig:Fig1}C-D**) and maintainability and readability of code and documentation. 
+
+The nature of the project required additional steps beyond peer review of analytical code (**Figure {@fig:Fig1}D**).
+Because multiple contributors were working on the code base and months would sometimes pass between updates to specific analysis modules, it was important to maintain a consistent software development environment to ensure that code could be revised. 
```

Comment:
```suggestion
Since multiple contributors were working on the code base and months would sometimes pass between updates to specific analysis modules, it was important to maintain a consistent software development environment to ensure that code could be revised. 
```